release.json

```diff
@@ -22,6 +22,30 @@
         "WINDOWS_DDNPM_SHASUM": "61e13fccdca7bf6a356e7ef552a7845316b9905b5eee264811982e3034a3bb84",
         "SECURITY_AGENT_POLICIES_VERSION": "master"
     },
+    "6.28.1-rc.1": {
+        "INTEGRATIONS_CORE_VERSION": "7.28.1",
```

Comment:
Have we verified the tag is in place in the integrations-core repo?

Reply:
Yeah: https://github.com/DataDog/integrations-core/tree/7.28.1